src/collections/_documentation/clients/javascript/config.md

```diff
@@ -43,7 +43,7 @@ Those configuration options are documented below:
   }
   ```
 
-  Can also be defined with `Raven.setRelease('721e41770371db95eee98ca2707686226b993eda')`.
+  Can also be defined with `Raven.setRelease('721e41770371db95eee98ca2707686226b993eda')`. See the [releases](https://docs.sentry.io/workflow/releases/?platform=javascript) page for more information about releases.
```

Comment:
Can you change this to use a less hardcoded link? You should find examples for this with `git grep '{% link'`